index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Page Layout</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="header">
+    <div class="">
+      <nav class="navigation">
+        <p class=" header-logo-text">Header Logo</p>
+        <ul class="nav-menue">
+          <li>
+            <a class="navigation-text" href="">header link one</a>
+          </li>
+          <li>
+            <a class="navigation-text" href="">header link two</a>
+          </li>
+          <li><a class="navigation-text" href=""> header link three </a></li>
+        </ul>
+      </nav>
+      <div class="header-mid-section">
+        <div class="header-mainText-section">
+          <p class="header-mainText">This website is awesome</p>
+          <p class="header-secondary-text">
+            This website has some subtext that goes here under the main title.
+            It's smaller front and the color is lower contrast
+          </p>
+          <button class="header-signup-button">Sign up</button>
+        </div>
+        <div class="header-img-placeholder">this is a placeholder for an image</div>
+      </div>
+    </div>
+  </div>
+
+  <div class="info-section">
+    <p class="info-header-text">Some random information</p>
+    <div class=" info-card">
+      <div class="info-content">
+        <div class="info-box">
```

Comment:
```html
<div class="info-content">
        <div class="info-box">
```
to 
```html
<div class="info-box">
     <div class="info-content">

````

this would be more relevant, as a box can have content but content can't have box right ?